assets/tvs/multitv/includes/multitv.class.php

```diff
@@ -283,6 +283,8 @@ function prepareSettings($settings)
             $this->configuration['displayLengthMenutext'][] = ($displayLength != -1) ? $displayLength : $this->language['all'];
         }
         $this->configuration['editBoxWidth'] = isset($settings['configuration']['editBoxWidth']) ? $settings['configuration']['editBoxWidth'] : '';
+        $this->configuration['css'] = isset($settings['configuration']['css']) ? $settings['configuration']['css'] : '';
```

Comment:
My first idea was a bit overcomplicated:
 
Could you prepare the css array here from the comma separated list?

```
(isset($settings['configuration']['css']) && !empty($settings['configuration']['css'])) ? explode(',', $this->configuration['css']) : array();
```

and remove the tvCSS things in the lines below. After that array_merge in line 838 should be the only thing needed.